pkg/operator/targetconfigcontroller/targetconfigcontroller_test.go

```diff
@@ -88,14 +96,45 @@ func TestTargetConfigController(t *testing.T) {
 				u.FakeEtcdMemberWithoutServer(0),
 				u.FakeEtcdMemberWithoutServer(2),
 			},
+			etcdBackupSpec:    nil,
 			etcdMembersEnvVar: "1,3",
 		},
+		{
+			name: "Backup Var Test",
```

Comment:
this test case fails, since the static pod manifest does not contains the sidecar, not the backup vars, it depends on 

https://github.com/openshift/cluster-etcd-operator/pull/1304